spec/models/author_spec.rb

```diff
@@ -195,6 +195,16 @@
       expect(Publication.find(pub.id).pub_hash[:authorship].length).to eq(1)
     end
 
+    it 'does not create contrib or save publication when not needed' do
+      # assign a pub
+      subject.assign_pub(pub)
+      expect(subject.contributions.count).to eq(1)
+      # assign it again, pub should not change!
+      subject.assign_pub(pub)
+      expect(subject.contributions.count).to eq(1)
+      expect(pub).not_to be_changed
```

Comment:
`changed?` always returns false after a save, so I'm not sure what you're trying to test for here.

Reply:
interesting...didn't realize this - made an adjustment to the test (basically trying to confirm that pub does not get re-saved if we re-assign that publication, so that is causing unnecessary touches of the publication updated_at)